app/assets/javascripts/admin/util.js.erb

```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
   $('.datetimepicker').datetimepicker({
-    dateFormat: Spree.translations.date_picker,
+    dateFormat: "yy-mm-dd",
```

Comment:
Are we using this somewhere else? Shouldn't we modify the Spree setting in that case?